app/models/user.rb

```diff
@@ -9,4 +9,12 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  has_many :friend_requests
```

Comment:
The friend_requests are currently working as your friendship, but that shouldn't be the case.
This association should show only the friendships that have a `confirmed: false`.
When a user accepts a friendship request you just change the confirmed to true and it becomes a friend.

Reply:
We have refactored the association so that `:friend_requests` only return records where `confirmed: false`

Reply:
We have refactored the association so that `:friend_requests` only return records where `confirmed: false`